po/meson.build

```diff
@@ -1,18 +1,34 @@
-gettext_deps = []
+dep_intl = []
 if get_option('nls')
-  if host_machine.system() == 'linux' or host_machine.system() == 'gnu'
-    gettext_deps = []
-  elif host_machine.system() == 'freebsd' or host_machine.system() == 'openbsd' or host_machine.system() == 'dragonfly'
-    gettext_deps = [cc.find_library('intl', has_headers: ['libintl.h'], dirs: ['/usr/local/lib'], header_args: inc_args)]
-  elif host_machine.system() == 'netbsd'
-    gettext_deps = [cc.find_library('intl', has_headers: ['libintl.h'], dirs: ['/usr/pkg/lib'], header_args: inc_args)]
-  else # darwin, cygwin
-    gettext_deps = [cc.find_library('intl', has_headers: ['libintl.h'])]
+  inc_args = []
+  dep_intl = dependency('intl', required: false)
+  if not dep_intl.found()
+    if host_sys == 'freebsd' or host_sys == 'openbsd' or host_sys == 'dragonfly'
+      inc_args = ['-isystem/usr/local/include']
+      dep_intl = [cc.find_library(
+        'intl',
+        has_headers: ['libintl.h'],
+        dirs: ['/usr/local/lib'],
+        header_args: inc_args
+        )]
+    elif host_sys == 'netbsd'
+      inc_args = ['-isystem/usr/pkg/include']
+      dep_intl = [cc.find_library(
+        'intl',
+        has_headers: ['libintl.h'],
+        dirs: ['/usr/pkg/lib'],
+        header_args: inc_args
+        )]
+    else # darwin, cygwin
+      dep_intl = [cc.find_library(
+        'intl',
+        has_headers: ['libintl.h']
+        )]
+    endif
   endif
-
+  add_project_arguments(inc_args, language : 'c')
```

Comment:
@eli-schwartz Here I need to check if dep_intl.found() or if the array dep_intl is not empty.